chainerrl/agents/iqn.py

```diff
@@ -125,6 +126,67 @@ def evaluate_with_quantile_thresholds(taus):
         return evaluate_with_quantile_thresholds
 
 
+class StatelessRecurrentImplicitQuantileQFunction(
```

Comment:
I don't think `StatelessRecurrentImplicitQuantileQFunction` inheriting `ImplicitQuantileQFunction` is a good idea because they have completely different interfaces. But it is true that we can move the content of `evaluate_with_quantile_thresholds` outside these classes so it can be reused. I will.